pyo3-ffi/src/object.rs

```diff
@@ -28,21 +39,39 @@ pub const PyObject_HEAD_INIT: PyObject = PyObject {
 // skipped PyObject_VAR_HEAD
 // skipped Py_INVALID_SIZE
 
+#[repr(C)]
+#[derive(Copy, Clone)]
+#[cfg(Py_3_12)]
+pub union __pyo3_object_ob_refcnt {
```

Comment:
I suspect the union is anonymous in the C header, but can't we give it a nicer name here, e.g. `PyObjectRefcnt` or something like that?

Reply:
Yes it's anonymous in the C header. I quite wanted to give it a name which implied that PyO3 had been forced to invent a name. I'll go for `PyObjectObRefcnt` (as the field is `ob_refcnt`), and add a docstring explaining this type is invented by PyO3.